inlong-manager/manager-client-examples/src/test/java/org/apache/inlong/manager/client/File2HBaseExample.java

```diff
@@ -0,0 +1,150 @@
+            streamBuilder.fields(createStreamFields());
+            streamBuilder.source(createAgentFileSource());
+            streamBuilder.sink(createHBaseSink());
+            streamBuilder.initOrUpdate();
+            // start group
+            InlongGroupContext inlongGroupContext = group.init();
+            Assert.notNull(inlongGroupContext);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Test
+    public void testStopGroup() {
+        ClientConfiguration configuration = new ClientConfiguration();
+        configuration.setWriteTimeout(10);
+        configuration.setReadTimeout(10);
+        configuration.setConnectTimeout(10);
+        configuration.setTimeUnit(TimeUnit.SECONDS);
+        configuration.setAuthentication(super.getInlongAuth());
+        InlongClient inlongClient = InlongClient.create(super.getServiceUrl(), configuration);
+        InlongGroupInfo groupInfo = createGroupInfo();
+        try {
+            InlongGroup group = inlongClient.forGroup(groupInfo);
+            InlongGroupContext groupContext = group.delete();
+            Assert.notNull(groupContext);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    private FileSource createAgentFileSource() {
+        FileSource fileSource = new FileSource();
+        fileSource.setSourceName("{source.name}");
+        fileSource.setAgentIp("{agent.ip}");
+        fileSource.setPattern("/a/b/*.txt");
+        fileSource.setTimeOffset("-1h");
+        return fileSource;
+    }
+
+    private List<StreamField> createStreamFields() {
+        List<StreamField> streamFieldList = Lists.newArrayList();
+        streamFieldList.add(new StreamField(0, FieldType.STRING.toString(), "name", null, null));
+        streamFieldList.add(new StreamField(1, FieldType.INT.toString(), "age", null, null));
+        streamFieldList.add(new StreamField(2, FieldType.DECIMAL.toString(), "score", null, null));
+        return streamFieldList;
+    }
+
+    /**
+     * Create iceberg sink
+     */
+    public HBaseSink createHBaseSink() throws Exception {
+        HBaseSink sink = new HBaseSink();
+
+        sink.setSinkName("{sink.name}");
+        sink.setNamespace("{db.name}");
+        sink.setTableName("{table.name}");
+        sink.setZkQuorum("{ip:port}");
+        sink.setZkNodeParent("{zk.node.path}");
+        sink.setRowKey("{rowkey}");
+
+        final SinkField field1 = new SinkField(0, FieldType.INT.toString(), "age", FieldType.INT.toString(), "age");
+        final SinkField field2 = new SinkField(1, FieldType.STRING.toString(), "name", FieldType.STRING.toString(),
+                "name");
+        final SinkField field3 = new SinkField(2, FieldType.DECIMAL.toString(), "score", FieldType.DECIMAL.toString(),
+                "score");
+
+        // field ext param
+        final ObjectMapper mapper = new ObjectMapper();
```

Comment:
Suggest using a global instance of ObjectMapper, for example, extract this instance into the super class `BaseExample`.